softlayer/resource_softlayer_virtual_guest_test.go

```diff
@@ -198,36 +201,76 @@ func testAccCheckSoftLayerVirtualGuestExists(n string, guest *datatypes.Virtual_
 	}
 }
 
+func CheckStringSet(n string, name string, set []string) resource.TestCheckFunc {
```

Comment:
`CheckStringSet` doesn't strictly have to be exported (but maybe it makes it more clear that we see it as a common utility func?)